subscription_platform/explores/daily_active_logical_subscriptions.explore.lkml

```diff
@@ -0,0 +1,27 @@
+include: "../views/daily_active_logical_subscriptions.view.lkml"
+include: "../views/logical_subscriptions.view.lkml"
+include: "../views/table_metadata.view.lkml"
+include: "/shared/views/countries.view.lkml"
+
+explore: daily_active_logical_subscriptions {
+
+  join: countries {
+    sql_on: ${daily_active_logical_subscriptions.subscription__country_code} = ${countries.code} ;;
+    type: left_outer
+    relationship: many_to_one
+  }
+
+  join: current_subscriptions {
+    from: logical_subscriptions
+    sql_on: ${daily_active_logical_subscriptions.subscription__id} = ${current_subscriptions.id} ;;
+    type: left_outer
+    relationship: many_to_one
+  }
+
+  join: table_metadata {
+    sql_on: ${table_metadata.table_name} = 'daily_active_logical_subscriptions_v1' ;;
+    type: left_outer
+    relationship: many_to_one
```

Comment:
@relud for similar joins in the VPN explores [you used `one_to_one` relationships](https://github.com/mozilla/looker-spoke-default/blob/2dba0c36d89dd160c1869ea1a16380c7a68a2a0f/mozilla_vpn/explores/subscriptions.explore.lkml#L16), with the explanation:
> Using a one_to_one relationship here, instead of the technically correct many_to_one, makes
> Looker understand that this join does not impact aggregation, which only works because this
> view does not contain any aggregates.

I didn't think `many_to_one` relationships impacted aggregations, just `one_to_many` and `many_to_many`. In what way were you seeing aggregations being impacted by a `many_to_one` relationship for such table metadata joins?

Reply:
i believe i was getting errors about not having a distinct id for sum measures when using a join labelled many_to_one instead of one_to_one. if you aren't getting errors, i think you're fine.